src/normal_form_game.jl

```diff
@@ -53,6 +53,61 @@ function Base.show(io::IO, player::Player)
     Base.print_array(io, player.payoff_array)
 end
 
+# delete_action
+
+"""
+    delete_action(player, action, player_idx)
```

Comment:
Make the default value explicit:

```suggestion
    delete_action(player, action[, player_idx=1])
```